### ISSUE TITLE
PSR12/Squiz/PEAR standards all error on promoted properties with docblocks

### ISSUE THREAD

**Issue by Arslanoov** [squizlabs/PHP_CodeSniffer#3342]
**Describe the bug**
Promoted properties without PHPDoc was fixed in 3.6.0, but with PHPDoc it still gives an error (in 3.6.0):
Multi-line function declaration not indented correctly; expected 8 spaces but found 0

**Code sample**
```php
use Symfony\Component\Validator\Constraints as Assert;

final class Command
{
    public function __construct(
        /**
         * @var string
         * @Assert\NotBlank()
         */
        public string $username
    )
    {
    }
}
```

**Custom ruleset**
```xml
<?xml version="1.0" encoding="UTF-8"?>

<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <arg name="basepath" value="."/>
    <arg name="cache" value=".phpcs-cache"/>
    <arg name="colors"/>
    <arg name="extensions" value="php"/>

    <rule ref="PSR12"/>

    <file>src/</file>
</ruleset>
```

**To reproduce**
You can try to build this project:
https://github.com/Arslanoov/messenger-api
1. git clone https://github.com/Arslanoov/messenger-api.git
2. make init
3. make install-deps
4. make cs

Or create a class with promoted properties and PHPDoc
```
ERROR | [x] Multi-line function declaration not indented correctly; expected 8 spaces but found 0
```

**Expected behavior**
No error on promoted properties with PHPDoc

**Versions (please complete the following information):**
 - OS: Ubuntu 20.04
 - PHP: 8.0
 - PHPCS: 3.6.0
 - Standard: PSR12

**Comment by Arslanoov** [comment]
With PHPDoc
![with](https://user-images.githubusercontent.com/51407990/117626014-16271880-b190-11eb-81cc-13686fd7972a.png)

Without PHPDoc
![withour](https://user-images.githubusercontent.com/51407990/117626028-1c1cf980-b190-11eb-8a0d-5a5a24c6ba91.png)

**Comment by jrfnl** [comment]
Related to #3227, #3210, #3323

**Comment by jrfnl** [comment]
PR #3343 should fix this. Testing appreciated.
